NEWS.txt

```diff
@@ -3,249 +3,183 @@ Release Notes
- - In MRG module:
-
-   - Replaced method ``multinomial_wo_replacement()`` with new method ``choice()``
-   - Random generator now tries to infer the broadcast pattern of its output
-
- - New pooling interface
- - Pooling parameters can change at run time
- - Moved ``softsign`` out of sandbox to ``theano.tensor.nnet.softsign``
- - Using floatX dtype when converting empty list/tuple
- - ``Roll`` make the shift be modulo the size of the axis we roll on
- - ``round()`` default to the same as NumPy: half_to_even
+ - Changed ``grad()`` method to ``L_op()`` in ops that need the outputs to compute gradient
+ - Merged duplicated diagonal functions into two ops: ``ExtractDiag`` (extract a diagonal to a vector),
+   and ``AllocDiag`` (set a vector as a diagonal of an empty array)
+ - Replaced ``MultinomialWOReplacementFromUniform`` with ``ChoiceFromUniform``
+
+ - Removed or deprecated Theano flags:
+
+   - ``cublas.lib``
+   - ``cuda.enabled``
+   - ``enable_initial_driver_test``
+   - ``gpuarray.sync``
+   - ``home``
+   - ``lib.cnmem``
+   - ``nvcc.*`` flags
+   - ``pycuda.init``
 
 Convolution updates:
- - Support of full and half modes for 2D and 3D convolutions including in ``conv3d2d``
- - Allowed pooling of empty batch
- - Implement ``conv2d_transpose`` convenience function
- - Multi-cores convolution and pooling on CPU
- - New abstract 3d convolution interface similar to the 2d convolution interface
- - Dilated convolution
-
+ - Extended Theano flag ``dnn.enabled`` with new option ``no_check`` to help speed up cuDNN importation
+ - Implemented separable convolutions
+ - Implemented grouped convolutions
 
 GPU:
- - cuDNN: support versoin 5.1 and wrap batch normalization (2d and 3d) and RNN functions
- - Multiple-GPU, synchrone update (via platoon, use NCCL)
- - Gemv(matrix-vector product) speed up for special shape
- - cublas gemv workaround when we reduce on an axis with a dimensions size of 0
- - Warn user that some cuDNN algorithms may produce unexpected results in certain environments
-   for convolution backward filter operations
- - ``GPUMultinomialFromUniform`` op now supports multiple dtypes
- - Support for ``MaxAndArgMax`` for some axis combination
- - Support for solve (using cusolver), erfinv and erfcinv
- - Implemented ``GpuAdvancedSubtensor``
+ - Prevent GPU initialization when not required
+ - Added disk caching option for kernels
+ - Added method ``my_theano_function.sync_shared()`` to help synchronize GPU Theano functions
+ - Added useful stats for GPU in profile mode
+ - Added Cholesky op based on ``cusolver`` backend
+ - Added GPU ops based on `magma library <http://icl.cs.utk.edu/magma/software/>`_:
+   SVD, matrix inverse, QR, cholesky and eigh
+ - Added ``GpuAdvancedIncSubtensor``
+ - Added ``GpuCublasTriangularSolve``
+ - Added atomic addition and exchange for ``long long`` values in ``GpuAdvancedIncSubtensor1_dev20``
+ - Fixed C code for log gamma function, now supporting all types except complex types.
```

Comment:
what this mean? If we had a crash, just tell that now we support log gamma on more dtype.